src/libsync/propagateupload.cpp

```diff
@@ -268,6 +270,8 @@ void PropagateUploadFileQNAM::slotComputeTransmissionChecksum(const QByteArray&
 
     connect(computeChecksum, SIGNAL(done(QByteArray,QByteArray)),
             SLOT(slotStartUpload(QByteArray,QByteArray)));
+    connect(computeChecksum, SIGNAL(done(QByteArray,QByteArray)),
+            computeChecksum, SLOT(deleteLater()));
     const QString filePath = _propagator->getFilePath(_item->_file);
```

Comment:
Do we have a double free now? Windows is more picky about that, in case Qt is not taking care of this... which I don't know.